entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+set -x
+# Variable precedence is environment variable defined, then command line supplied,
+# and finally default values.
+SERVER_IP="${SERVER_IP:-${1:-127.0.0.1}}"
+PASSPHRASE="${PASSPHRASE:-${2:-Password1}}"
```

Comment:
This is a fancy way to set default values in bash. It's nested so in the example of PASSPHRASE it first checks if PASSPHRASE is defined (which it will be in docker-compose launches). If it isn't it will check the environment variables with 2 corresponding to $2. Finally if it wasn't supplied in the command line arguments the default values will be picked up.